integration-tests/it-consumer/src/main/resources/microservice.yaml

```diff
@@ -16,4 +16,17 @@
 ## ---------------------------------------------------------------------------
 
 service_description:
+  environment: development
```

Comment:
why need this parameter?

Reply:
Make local test simpler. I added some test cases, and need to delete sc's data. Add this make local development easier.